src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.NonBrowser.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
```

Comment:
Should be called something like `.EventSource.xml` instead

Reply:
I was going for having the name of the file to indicate build configuration inclusion as opposed to actual contents of it, which is what we've done in the rest of the repo for similar file exclusions/inclusions. That said, if you feel strongly about the name being wrong then I can rename it.

Reply:
I don't feel strongly about this but mixing features and platforms is always problematic